components/gitpod-db/src/typeorm/workspace-db-impl.ts

```diff
@@ -476,7 +476,12 @@ export class TypeORMWorkspaceDBImpl extends TransactionalDBImpl<WorkspaceDB> imp
         const sessions = (await workspaceInstanceRepo
             .createQueryBuilder("wsi")
             .leftJoinAndMapOne("wsi.workspace", DBWorkspace, "ws", "ws.id = wsi.workspaceId")
-            .leftJoinAndMapOne("wsi.metrics", DBWorkspaceInstanceMetrics, "wsim", "wsim.instanceId = wsi.id")
+            .leftJoinAndMapOne(
+                "wsi.metrics",
+                DBWorkspaceInstanceMetrics,
+                "wsim",
+                "wsim.instanceId COLLATE utf8mb4_general_ci = wsi.id COLLATE utf8mb4_general_ci",
```

Comment:
The 2nd `COLLATE` is superfluous.

Reply:
But also... why not leave it as-is, looking at the mess in some of the tables schemas... :melting_face:

Reply:
> But also... why not leave it as-is, looking at the mess in some of the tables schemas... 🫠

I did it because of the original request XD "Maybe we can fix that in the query itself (there is a way to force a collation on the fly)."